components/control/Button/Button.css

```
@@ -0,0 +1,195 @@
+  --_padding-inline: var(--diamond-button-padding-inline);
+  border-radius: var(--diamond-button-border-radius);
+
+  font: inherit;
+  letter-spacing: inherit;
+  line-height: 1.5;
+}
+
+button,
+diamond-button a,
+input[type='button'],
+input[type='submit'],
+input[type='reset'],
+input[type='file']::file-selector-button {
+  align-items: center;
+  background: var(--_background);
+  border: var(--diamond-border);
+  border-color: var(--_border-color);
+  color: var(--_color);
+  cursor: pointer;
+  display: inline-flex;
+  font-size: var(--_size, 1rem);
+  font-weight: var(--diamond-button-font-weight);
+  gap: var(--diamond-button-gap);
+  justify-content: center;
+  padding-block: var(--_padding-block);
+  padding-inline: var(--_padding-inline);
+  -webkit-tap-highlight-color: transparent;
+  text-align: center;
+  text-shadow: var(--diamond-button-text-shadow);
+  touch-action: manipulation;
+  -webkit-touch-callout: none;
+  transition:
+    background var(--diamond-transition),
+    color var(--diamond-transition),
+    border-color var(--diamond-transition),
+    outline-offset var(--diamond-transition);
+  user-select: none;
+
+  @media (hover: hover) {
+    &:hover {
+      --_background: var(--_background-hover);
+      --_border-color: var(--_border-color-hover);
+      --_color: var(--_color-hover);
+    }
+  }
+
+  &:not(:active):focus-visible {
+    outline-offset: var(--diamond-button-outline-offset);
+  }
+
+  &[type='button'] {
+    appearance: none;
+  }
+
+  &[disabled] {
+    --_background: var(--_background-disabled);
+    --_color: var(--_color-disabled);
+    --_border-color: var(--_border-color-disabled);
+    cursor: not-allowed;
+  }
+}
+
+input[type='file'] {
+  background-color: var(--diamond-button-file-background);
+  inline-size: 100%;
+  max-inline-size: max-content;
+
+  &::file-selector-button {
+    appearance: none;
+    margin-inline-end: var(--diamond-spacing);
+  }
+}
+
+button,
+diamond-button a,
+input[type='button'],
+input[type='submit'],
+input[type='reset'] {
+  svg {
+    block-size: var(--diamond-button-icon-size);
+    fill: none;
+    filter: drop-shadow(var(--diamond-button-text-shadow));
+
+    flex-shrink: 0;
+    inline-size: var(--diamond-button-icon-size);
+    stroke: currentColor;
+    stroke-linecap: round;
+    stroke-linejoin: round;
+  }
+}
+
+/**
+ * diamond-button specific styles
+ */
+diamond-button a {
+  color: var(--_color);
+  text-decoration: none;
+}
+
+diamond-button :where(button),
+diamond-button :where(input[type='button']),
+diamond-button :where(input[type='submit']),
+diamond-button :where(input[type='reset']),
+diamond-button :where(input[type='file']),
+diamond-button :where(input[type='file'])::file-selector-button,
+diamond-button a {
+  diamond-button[size='sm'] & {
+    --_size: 0.875rem;
+  }
+
+  diamond-button[size='lg'] & {
+    --_size: 1.25rem;
+  }
+
+  diamond-button[width='full-width'] & {
+    width: 100%;
+  }
+
+  @media (max-width: 480px) {
```

```suggestion
  @media (width < 480px) {
```

480 includes 480 so all the min width ones will flip at 480, but this one won't. Relative media queries are less ambiguous